dlang.org.ddoc

```diff
@@ -241,6 +241,13 @@ PHOBOSSRC=$(SPANC phobos_src, $(AHTTPS github.com/D-Programming-Language/phobos/
 _=
 
 RED=$(SPANC red, $0)
+PREFIX_DOT=.$1$(PREFIX_DOT $+)
+PREFIX_DOT_SKIP=$(PREFIX_DOT $+)
+PREFIX_UNDERSCORE=_$1$(PREFIX_UNDERSCORE $+)
+PREFIX_UNDERSCORE_SKIP=$(PREFIX_UNDERSCORE $+)
+PREFIX_UNDERSCORE_SKIP2=$(PREFIX_UNDERSCORE_SKIP $+)
```

Comment:
See @adamdruppe 's recent PR for simple macros that prefix with dots, underscores etc.